ios/Inji.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -28,5 +29,5 @@
       }
     }
   ],
-  "version" : 2
+  "version" : 3
```

Comment:
please make it version 2. CI build may fail otherwise.

Reply:
Done